src/modules/governance/models/governance.proposal.model.ts

```diff
@@ -0,0 +1,54 @@
+import { Field, Int, ObjectType } from '@nestjs/graphql';
+import { GovernanceAction } from './governance.action.model';
+import { EsdtTokenPaymentModel } from '../../tokens/models/esdt.token.payment.model';
+import { ProposalVotes } from './proposal.votes.model'; //Assuming you will create GovernanceAction model separately
+
+@ObjectType()
+export class Description {
+    @Field()
+    title: string;
+    @Field()
+    hash: string;
+    @Field(() => Int)
+    strapiId: number;
+
+    constructor(init: Partial<Description>) {
+        Object.assign(this, init);
+    }
+}
+
+@ObjectType()
+export class GovernanceProposal {
+    @Field()
+    contractAddress: string;
+    @Field()
+    proposalId: number;
+    @Field()
+    proposer: string;
+    @Field(() => [GovernanceAction])
+    actions: GovernanceAction[];
+    @Field( () => Description)
+    description: Description;
+    @Field(() => EsdtTokenPaymentModel)
+    feePayment: EsdtTokenPaymentModel;
+    @Field()
+    minimumQuorum: string;
```

Comment:
number maybe?

Reply:
quorum is expressed in bigNumber as it will be total energy or total token supply

Reply:
understood